lib/store_web/templates/layout/primary-nav.html.leex

```diff
@@ -37,7 +37,15 @@
     </li>
   </ul>
 
-  <ul>
+
+   <ul>
+	<li><a href="https://youtube.com/user/elementaryproject" target="_blank" rel="noopener" title="Youtube">
+        <svg fill="#ffffff" xmlns="http://www.w3.org/2000/svg"  viewBox="0 6 46 40" height="550px"><path d="M44.9,14.5c-0.4-2.2-2.3-3.8-4.5-4.3C37.1,9.5,31,9,24.4,9c-6.6,0-12.8,0.5-16.1,1.2c-2.2,0.5-4.1,2-4.5,4.3C3.4,17,3,20.5,3,25s0.4,8,0.9,10.5c0.4,2.2,2.3,3.8,4.5,4.3c3.5,0.7,9.5,1.2,16.1,1.2s12.6-0.5,16.1-1.2c2.2-0.5,4.1-2,4.5-4.3c0.4-2.5,0.9-6.1,1-10.5C45.9,20.5,45.4,17,44.9,14.5z M19,32V18l12.2,7L19,32z"/></svg>
```

Comment:
It looks like for the other SVGs we don't use an inline fill color, so it can be set by CSS.

```suggestion
        <svg xmlns="http://www.w3.org/2000/svg"  viewBox="0 6 46 40" height="550px"><path d="M44.9,14.5c-0.4-2.2-2.3-3.8-4.5-4.3C37.1,9.5,31,9,24.4,9c-6.6,0-12.8,0.5-16.1,1.2c-2.2,0.5-4.1,2-4.5,4.3C3.4,17,3,20.5,3,25s0.4,8,0.9,10.5c0.4,2.2,2.3,3.8,4.5,4.3c3.5,0.7,9.5,1.2,16.1,1.2s12.6-0.5,16.1-1.2c2.2-0.5,4.1-2,4.5-4.3c0.4-2.5,0.9-6.1,1-10.5C45.9,20.5,45.4,17,44.9,14.5z M19,32V18l12.2,7L19,32z"/></svg>
```

Reply:
Yeah am sorry, i wasn't able to find the css file so i was hoping everything was the same.
Thanks for the tweaks.
Is the rest alight though?